sunpy/coordinates/ephemeris.py

```diff
@@ -282,46 +287,97 @@ def get_horizons_coord(body, time='now', id_type=None, *, include_velocity=False
     ...                    time={{'start': '2020-12-01',
     ...                           'stop': '2020-12-02',
     ...                           'step': '12'}})  # doctest: +REMOTE_DATA
-    INFO: Obtained JPL HORIZONS location for Solar Orbiter (spacecraft) (-144 [sunpy.coordinates.ephemeris]
+    INFO: Obtained JPL HORIZONS location for Solar Orbiter (spacecraft) (-144) [sunpy.coordinates.ephemeris]
     ...
     """
-    # Import here so that astroquery is not a module-level dependency
-    import astroquery
-    from astroquery.jplhorizons import Horizons
-
-    if id_type is None and version.parse(astroquery.__version__) < version.parse('0.4.4'):
-        # For older versions of astroquery retain default behaviour of this function
-        # if id_type isn't manually specified.
-        id_type = 'majorbody'
+    args = {
+        'EPHEM_TYPE': 'VECTORS',
+        'OUT_UNITS': 'AU-D',
+        'CENTER': '500@10',
```

Comment:
Maybe add comments to say what AU-D and 500@10 mean?

Reply:
Added